labs/firepower-restapi-101/1.md

```diff
@@ -1,46 +1,53 @@
+
+FMC 6.1 supplements the FMC web-based GUI with the first release of FMC REST APIs. These APIs enable administrators to integrate FMC with existing applications, and they also enable workflow automation.
 
 ![Figure: FMC REST API example setup](/posts/files/firepower-restapi-101/assets/images/FMC1.png)
 
-## Enable FMC REST API using FMC UI
-To use REST API in FMC, make sure REST API is turned on in your own FMC setup; **By default REST API is turned on FMC**.
-If you are using DevNet sandbox  https://fmcrestapisandbox.cisco.com/api/api-explorer/, REST API is already enabled by the administrator.
+## Enabling the FMC REST API
+
+The FMC REST API must be enabled in your FMC setup, and it is enabled by default. It is also enabled by default in [the DevNet sandbox](https://fmcrestapisandbox.cisco.com/api/api-explorer/).
+
+If you need to manually enable FMC REST API, follow this procedure.
+
+1. In your browser, navigate to FMC UI. The URL will be `https://*FMC_hostname_or_address*`.
 
-**Note on obtaining username/password for FMC sandbox:**
-```shell
-The API Explorer URL is publically available. However, in order to login to the FMC, users must first be allocated credentials. This is done by reserving the lab for however long you require access i.e. hit the "RESERVE" button in the upper right of the Firepower Management Center sandbox page which can be access on this URL: https://developer.cisco.com/site/devnet/sandbox/docs/index.gsp#security/overview
+2. From the **System** menu in the upper left corner, select the **Configuration** tab.
 
-Select Firepower Management Center from the above page.
+3. On the **Configuration** tab, select **REST API Preferences** and ensure that **Enable REST API** is selected.
 
-You will require cisco.com credentials to access the reservation page from the above URL
+![Figure: FMC UI API setup](/assets/images/fmcui.png)
 
-Once reserved, a username/password will be sent to you via email and also displayed in the output window on this page. These credentials are valid for the duration of your reservation. You will not be able to login to the API Explorer once the reservation as ended.
+### Obtaining a username and password for FMC Sandbox
 
-If further access is required, just reserve the lab again and new credentials will be sent out.
+To log in to the FMC Sandbox, users must reserve lab time to obtain credentials.
 
-As this is an always on lab and resources are shared, users are allocated read only rights.
-```
+1. Navigate to [the DevNet Sandbox documentation](https://developer.cisco.com/site/devnet/sandbox/docs/index.gsp#security/overview) and log in with your cisco.com credentials.
+2. Click **Firepower Management Center**.
+3. On the DevNet Sandbox page, click the **RESERVE** button.
+4. In the Reserve dialog, accept the defaults and click **Reserve**. The Sandbox reservation setup process begins, and will take several minutes to complete.
```

Comment:
It was about five minutes for me, and that's also what the Sandbox says. I wonder if those are live estimates based on realtime traffic? Anyway, give it a try and see if this can be changed to "about five minutes to complete."